b2share/modules/schemas/views.py

```diff
@@ -139,12 +183,128 @@ def get(self, community_schema):
         return community_schema
 
 
+class BlockSchemaListResource(ContentNegotiatedMethodView):
+    view_name = 'block_schema_list'
+
+    def __init__(self, **kwargs):
+        """Constructor."""
+        super(BlockSchemaListResource, self).__init__(
+            method_serializers={
```

Comment:
The serializers are the same so we can use ["serializers" as above](https://github.com/EUDAT-B2SHARE/b2share/pull/986/files#diff-48b91218db4176f52bd6e00d476fd5b1L79)